apps/amo/celery.py

```diff
@@ -0,0 +1,33 @@
+from __future__ import absolute_import
+
+import commonware.log
+from celery import Celery
+from celery.signals import task_failure
+from django.conf import settings
+
+
+log = commonware.log.getLogger('z.celery')
+
+
+app = Celery('olympia')
+task = app.task
+
+app.config_from_object('django.conf:settings')
+app.autodiscover_tasks(settings.INSTALLED_APPS)
+
+
+@task_failure.connect
+def process_failure_signal(exception, traceback, sender, task_id,
+                           signal, args, kwargs, einfo, **kw):
+    exc_info = (type(exception), exception, traceback)
```

Comment:
This is a slightly modified version is what was previously imported from `celeryutils`. The rest of that module is no longer required.